src/pager/index.js

```diff
@@ -0,0 +1,118 @@
+import React, { Component, PropTypes } from 'react';
+import Icon from '../../infl-components/icon.jsx';
+import classnames from 'classnames';
+
+const MOVING_WINDOW_SIZE = 5;
+const EDGE_WINDOW_SIZE = 2;
+
+export default class Pager extends Component {
+
+  static propTypes = {
+    /**
+     * The current page number (starting at 1).
+     */
+    currentPage: PropTypes.number.isRequired,
+    /**
+     * The number of items that are displayed in each page.
+     */
+    perPage: PropTypes.number,
+    /**
+     * The total number of items that are available.
+     */
+    totalItemCount: PropTypes.number.isRequired,
+    /**
+     * This callback is called when the user navigates to a different page. It
+     * has a single argument which is the new page number.
+     */
+    onPageChangeCallback: PropTypes.func.isRequired,
+    /**
+     * The classname of the page element that is currenting displayed.
+     */
+    selectedClassName: PropTypes.string
+  };
+
+  static defaultProps = {
+    perPage: 10,
+    selectedClassName: ''
+  };
+
+  render() {
+    let {currentPage} = this.props;
```

Comment:
Looks like most of the let's in here should actually be const.

let is an indicator that a variable will be reassigned at some point.